### ISSUE TITLE
No window control buttons in the 0.4.4 release

### ISSUE THREAD

**Issue by Brn9hrd7** [nukeop/nuclear#238]
In the current release of Nuclear(0.4.4) there are no window control buttons(minimize, maximize & close).

**Comment by charjac** [comment]
in App.js
```jsx
  renderNavBar () {
    return (
      <Navbar className={styles.navbar}>
        <SearchBoxContainer />
        <Spacer />
        <Spacer />
        <WindowControls />
        {this.props.settings.framelessWindow && <WindowControls />}
      </Navbar>
    );
  }
```
this.props.settings.framelessWindow is undefined

**Comment by nukeop** [comment]
It's undefined until you select it in the settings. It should default to the value defined in const/settings.js.

**Comment by grount** [comment]
I'm still experiencing this, I see the issue is closed does it mean it will be solved in 0.4.5?

**Comment by nukeop** [comment]
This was fixed in the development build.
